### DISCUSSION TITLE
Training Printout Meaning

### DISCUSSION THREAD

**Discussion by htaoruan** [open-mmlab/mmocr#1852] [General]
In the log file, Epoch (train) [5] [10/10] lr: 1.0000e-05 eta: 0:00:00 time: 0.0481 data_ time: 0.0022 is printed out for each iteration ，time and data _ time What the means?

**Comment by gaotongxiao** [comment]
https://mmengine.readthedocs.io/en/latest/advanced_tutorials/logging.html?highlight=log

**Reply by htaoruan** [reply]
Thank you
